.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,86 @@
+name: Bug report
+description: File a bug report
+title: Describe the problem
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Your issue will be triaged by the RNW team according to [this process](https://github.com/microsoft/react-native-windows/wiki/Triage-Process)
+  - type: input
+    validations:
+      required: true
+    attributes:
+      label: CLI version
+      description: |
+        Run the command `npx react-native --version` in your terminal and copy the results here
+      value: "npx react-native --version"
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Environment
+      description: |
+        Run the command `npx react-native info` in your terminal and copy the results here
+      value: "npx react-native info"
+      render: markdown
+  - type: dropdown
+    attributes:
+      label: Target Platform Version
+      description: What SDK version are you building for?
+      options:
+        - "10.0.15063"
+        - "10.0.16299"
+        - "10.0.19041"
+  - type: dropdown
+    attributes:
+      label: Target Device(s)
+      description: What device(s) are you targeting?
+      multiple: true
+      options:
+        - "Desktop"
+        - "Xbox"
+        - "HoloLens"
+  - type: dropdown
+    attributes:
+      label: Visual Studio Version
+      description: Which version of Visual Studio are you using?
+      options:
+        - "Visual Studio 2017"
+        - "Visual Studio 2019"
+  - type: dropdown
+    attributes:
+      label: Build Configuration
+      description: Which build configuration are you running?
+      options:
+        - "Debug"
+        - "DebugBundle"
+        - "Release"
+        - "ReleaseBundle"
+  - type: textarea
```

Comment:
I know from myself if I'm ready to file a bug I have this and the next field already mentally prepared to enter... Having to run a bunch of commands first in the form risks, as a bug filer I personally would prefer these to be the first fields to enter.

Wouldn't it also make triaging easier if this and the next one are at the top of the bug report?

Reply:
Good question. I'd definitely love to streamline the order to match what people prefer when filing. For this I matched what the previous form had. But yeah... what's your ideal top-down order? It's currently frontloaded with some required fields (version/environment). Although "repro steps" and "expected results" are also required but are broken up by all of these optional dropdowns. And I agree, those should probably be at the top.

So:
- Repro steps
- Expected results
- Version
- Environment
- ...

Like that?

(I may take the md->yaml change as is because it's the same order as before, but I'm going to be following up with a change to the other templates and we should reorder to be better as you've expected)

Oh, and FYI Adaptive Cards has adopted the new forms if you want to try them out and see what works well and what doesn't: https://github.com/microsoft/AdaptiveCards/issues.

Reply:
Okay, I updated the form's order. It now looks like this:

![image](https://user-images.githubusercontent.com/26607885/133863083-a8433fd9-2df1-4899-898a-3edcb654ea88.png)